CHANGELOG.md

```diff
@@ -1,3 +1,1038 @@
+<a name="1.6.0-rc.0"></a>
+# 1.6.0-rc.0 bracing-vortex (Tue Oct 25 2016)
```

Comment:
Easier? To be clear I am talking about displaying the date as `Tue Oct 25 2016` vs `2016-10-25`.

We've been using the `YYYY-MM-DD` format in previous releases. I find it easier to read (especially since `Oct 25 2016` is locale-specific - e.g. some (sane 😛) locales put the day before the month) and I guess most people don't care what day of week it was 😛

But it's just a slight preference of mine. Not a biggy. Leaving it up to you.
<sub>(But we should stick with the format we choose for future releases.)</sub>